databases/postgresql16-server/Portfile

```diff
@@ -4,6 +4,7 @@ PortSystem 1.0
 
 name                postgresql16-server
 version             16.0
+revision            1
```

Comment:
This doesn't alter the installed code, so does it require a revision bump?

Reply:
Wouldn't this change the plist that gets generated and packaged with the port?